input/fsh/Profiles clinical datapoints/TreatmentVariables.fsh

```diff
@@ -0,0 +1,449 @@
+* reasonReference only Reference (PrimaryBreastCancerCondition)
+
+Instance: AxillaSurgeryPatient147
+InstanceOf: AxillaSurgery 
+Description: "Example of a breast cancer patient who underwent surgery to the axilla"
+* status = ProcedureStatusCS#completed
+* code = SCT#178294003 "Axillary lymph nodes sampling"
+* subject = Reference(BreastCancerPatient147)
+* performedDateTime = "2022-02-12"
+* reasonReference = Reference(PrimaryBreastCancerPatient147)
+
+Mapping: AxillaSurgeryToICHOM
+Source:	AxillaSurgery
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: axilla-surgery-mapping
+Title: "Axilla surgery to ICHOM set"
+Description: "Mapping of the axilla surgery to the ICHOM breast cancer PCOM set." 
+* -> "Treatment"	
+* category -> "Treatment"
+* code -> "Surgery axilla"
+* performedDateTime -> "Surgery axilla date" 
+
+// AXILLARY CLEARANCE
+Profile: AxillaryClearance
+Parent: Procedure 
+Id: axillary-clearance
+Title: "Axillary clearance"
+Description: "Represents if the breast cancer patient received axillary clearance due to lymph node involvement after sentinel lymph node biopsy during the last year. This profile is in alignment with mCODE."
+* partOf only Reference(AxillaSurgery)
+* code = SCT#79544006 "Complete axillary lymphadenectomy"
+* subject only Reference(BreastCancerPatient)
+* performedDateTime	and complication MS
+* reasonReference only Reference (AxillaSurgery)
+
+Instance: AxillaryClearancePatient147
+InstanceOf: AxillaryClearance
+Description: "Example of a breast cancer patient who received axilla clearance"
+* status = ProcedureStatusCS#completed
+* code = SCT#79544006 "Complete axillary lymphadenectomy"
+* subject = Reference(BreastCancerPatient147)
+* performedDateTime = "2022-02-12"
+* reasonReference = Reference(AxillaSurgeryPatient147)
+
+Mapping: AxillaryClearanceToICHOM
+Source:	AxillaryClearance
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: axillary-clearance-mapping
+Title: "Axillary clearance to ICHOM set"
+Description: "Mapping of axillary clearance to the ICHOM breast cancer PCOM set" 	
+* -> "Treatment"
+* code -> "Axillary clearance"
+* performedDateTime -> "Axillary clearance date" 
+
+// RECONSTRUCTION
+Profile: ReconstructionSurgery
+Parent: Procedure 
+Id: reconstruction-surgery
+Title: "Reconstruction surgery"
+Description: "Represents if the breast cancer patient received reconstruction surgery during the last year. This profile is in alignment with mCODE."
+* code = SCT#33496007 "Reconstruction of breast" // found this code myself, is not specifically delayed
+* subject only Reference(BreastCancerPatient)
+* performedDateTime	and complication MS
+* reasonReference only Reference (PrimaryBreastCancerCondition)
+* bodySite MS // need codes for pre pectoral and sub-pectoral reconstruction
+* focalDevice MS // need codes for the different types of impants (direct, staged, autologous implants)
```

Comment:
In the excel file there is a list of codes defined in column M (direct implant, staged implant,...). I dont see this profiled. Would it be useful to have Device profile and use it here for focalDevice.manipulated? Then the profile for Device would list these codes as value set for Device.association (this element is new in FHIR R5, to indicate the current association of the device(implanted/explanted/...) . Since you are not using FHIR R5 this can be defined as extension to be already compliant witht he new FHIR version.

Reply:
I think we can get by with just using terminology here without increasing the scope to include an additional resource. Managing a separate Device resource - unless we include a contained one - might not be everyone's cup of tea.

Reply:
I changed it to the element usedCode because the reference to a device (manipulated) is required (1..1) so it doesn't really fit. The example VS of the usedCode element also contains implants, so I think it's a fair fit.